klasses/models.py

```diff
@@ -0,0 +1,38 @@
+"""Models for klasses"""
+from django.db.models.base import Model
+from django.db.models.fields import (
+    DateTimeField,
+    DecimalField,
+    TextField,
+    IntegerField,
+)
+from django.db.models.fields.related import ForeignKey
+
+
+class Bootcamp(Model):
+    """
+    A bootcamp
+    """
+    title = TextField()
+
+
+class Klass(Model):
+    """
+    A class within a bootcamp
+    """
+    bootcamp = ForeignKey(Bootcamp)
+    klass_id = IntegerField()
+    price = DecimalField(max_digits=20, decimal_places=2)
+
```

Comment:
We are also building a dashboard for them in redash where ordering by class_start_date would be useful.